build.gradle

```diff
@@ -292,6 +292,8 @@ project(':cruise-control') {
     testImplementation 'org.bouncycastle:bcpkix-jdk15on:1.70'
     testImplementation 'org.apache.kerby:kerb-simplekdc:2.0.1'
     testImplementation 'com.jayway.jsonpath:json-path:2.7.0'
+    testImplementation 'org.powermock:powermock-module-junit4:2.0.9'
+    testImplementation 'org.powermock:powermock-api-mockito2:2.0.9'
```

Comment:
CC has been using easymock rather than mockito. Can we use the following dependency here -- and do the necessary updates on the test code to use easymock rather than mockito:
```
testImplementation 'org.powermock:powermock-api-easymock:2.0.9'
```